dxr/app.py

```diff
@@ -162,6 +164,27 @@ def _tree_tuples(trees, tree, query_text, is_case_sensitive):
              description)
             for t, description in trees.iteritems()]
 
+@dxr_blueprint.route('/<tree>/raw/<path:path>')
+def send_raw(tree, path):
+    """Send raw data at path from tree, for binary things like images."""
+    query = {
+            'filter': {
```

Comment:
We can outdent all these lines so the final } is aligned with "query". See some of the other JSON-ish places in query.py for examples. (Now I'm nitpicking, so you know your PR is in good shape! :-))